contrib/screener-api/go.mod

```diff
@@ -39,7 +39,6 @@ require (
 require (
 	dario.cat/mergo v1.0.0 // indirect
 	github.com/Azure/go-ansiterm v0.0.0-20210617225240-d185dfc1b5a1 // indirect
-	github.com/BurntSushi/toml v1.4.0 // indirect
```

Comment:
going forward if you're seeing these kinds of things, you can run [`make tidy`](https://github.com/synapsecns/sanguine/blob/master/make/repo.Makefile#L29-L31) from the repo root.

I've done this for you in aba64a79c79d30a8502a1213274d893f75b0124d

going forward the `needs-go-generate`... label is a hint you need to do this particularly this one: https://github.com/synapsecns/sanguine/pull/2933#event-13678590298) the other was caused by https://github.com/synapsecns/sanguine/pull/2933#issuecomment-2278707844